web-app-aspnet/README.md

```diff
@@ -24,8 +24,8 @@ The web app in this scenario has been created using the ASP.NET Core 6.0 Razor t
 
 ## Prerequisites
 
-- An Azure Active Directory (Azure AD) tenant. You can [open an Azure account for free](https://azure.microsoft.com/free) to get an Azure AD instance.
-- [.NET 6.0 SDK](https://dotnet.microsoft.com/download/dotnet/6.0)
+* An Azure account with an active subscription. If you don't already have one, [Create an account for free](https://azure.microsoft.com/free/?WT.mc_id=A261C142F).
+* [.NET Core SDK 6.0+](https://dotnet.microsoft.com/download)
```

Comment:
This should be .NET SDK. ["Core" is gone](https://github.com/dotnet/core/issues/6910) 😀

Reply:
Done!